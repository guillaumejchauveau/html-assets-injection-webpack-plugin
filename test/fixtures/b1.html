<!DOCTYPE html>
<p>b1</p>
