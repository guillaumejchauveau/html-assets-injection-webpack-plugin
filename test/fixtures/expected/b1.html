<!DOCTYPE html>
<link rel="stylesheet" href="./css/app.css"><p>b1</p>
<script src="./js/app.js"></script>
